From 3f1553afa4066b1a0dd790041f304612624f321f Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 11 Sep 2024 16:13:28 +0200
Subject: [PATCH 1121/1695] scripts: fix pycodestyle complain

fix following error:
scripts/sign_rproc_fw.py:96:32: E261 at least two spaces before inline
 comment.
scripts/sign_rproc_fw.py:97:31: E261 at least two spaces before inline
 comment.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9fe62722c2df82de3ed4d1472930c23c925965af scripts: fix pycodestyle complain]
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: If4793eeed760d0be9083827af62e6e5be60324ba
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/403798
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 scripts/sign_rproc_fw.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/sign_rproc_fw.py b/scripts/sign_rproc_fw.py
index d80818103..dfa9f7ffa 100755
--- a/scripts/sign_rproc_fw.py
+++ b/scripts/sign_rproc_fw.py
@@ -87,8 +87,8 @@ TLV_TYPES = {
 
 # Platform type definitions
 PLAT_TLV_TYPES = {
-        'BOOTADDR': 0x00010001, # boot address of the remoteproc firmware
-        'BOOTSEC': 0x00010002, # boot mode: secure or non-secure
+        'BOOTADDR': 0x00010001,  # boot address of the remoteproc firmware
+        'BOOTSEC': 0x00010002,   # boot mode: secure or non-secure
 }
 
 GENERIC_TLV_TYPE_RANGE = range(0x00000000, 0x00010000)
-- 
2.47.2

